R/connector-options.R

```diff
@@ -33,3 +33,9 @@ zephyr::create_option(
   default = FALSE,
   desc = "Add logs to the console as well as to the whirl log html files"
 )
+
+zephyr::create_option(
+  name = "default_ext",
+  default = "rds",
```

Comment:
For the option should be NULL by default. it allows the normal way of working for connector without having to change the option all the time. the option is  mostly useful for FS -> DBI or DBI -> FS. 
If NULL, then nothing change for the current way of working